components/oku-circ.vue

```diff
@@ -78,6 +78,18 @@ export default {
 $light-blue: #3C96D2;
 $red: #D0011B;
 
+// Width scale factor for Olin
+$olin-width: 0.053vw;
```

Comment:
Could go with `l-width`, `l-height` if the 2-letter abbreviations are too extreme ;)